### ISSUE TITLE
addGlobalIgnoredNamespace won't work with Annotations imported via use statement. Bug ?

### ISSUE THREAD

**Issue by re2bit** [doctrine/annotations#302]
I used the
```
AnnotationReader::addGlobalIgnoredNamespace('JMS');
``` 
to ignore Serializer Annotations. I expected this to ignore Annotations imported via use Statement as well. 
I don't know if this is a Bug or intended Behaviour.

Example: https://github.com/re2bit/annotations/commit/34f5be4fddc2396d9a427166bf9b249d82fd1800

**Comment by alcaeus** [comment]
Can you try to create a failing test case? That would greatly help with debugging this.

**Comment by re2bit** [comment]
done:
https://github.com/re2bit/annotations/commit/dfe8b0eb75bed234b55bbd615a86725e7650128f

In my Opinion this 2 Tests should behave the same way.
```
    /**
     * @group 45
     *
     * @runInSeparateProcess
     */
    public function testPropertyAnnotationIsIgnored()
    {
        $reader = $this->getReader();
        $ref = new \ReflectionClass(AnnotatedAtPropertyLevel::class);
        $reader::addGlobalIgnoredNamespace('SomePropertyAnnotationNamespace');
        self::assertEmpty($reader->getPropertyAnnotations($ref->getProperty('property')));
    }
    /**
     * @group 45
     *
     * @runInSeparateProcess
     */
    public function testPropertyAnnotationWithUseStatementIsIgnored()
    {
        $reader = $this->getReader();
        $ref = new \ReflectionClass(AnnotatedAtPropertyLevelWithUseStatement::class);
        $reader::addGlobalIgnoredNamespace('JMS');
        self::assertEmpty($reader->getPropertyAnnotations($ref->getProperty('property')));
    }
```

**Comment by alcaeus** [comment]
@re2bit can I take you closing this issue as confirmation that the PR works as expected?

**Comment by alcaeus** [comment]
Fixed in #244.
